api/hearings/models/hearings.enum.ts

```diff
@@ -48,7 +48,7 @@ export enum EXUIDisplayStatusEnum {
   FAILURE = 'FAILURE',
   CANCELLATION_REQUESTED = 'CANCELLATION REQUESTED',
   VACATED = 'VACATED',
-  AWAITING_ACTUALS = 'AWAITING ACTUALS',
+  AWAITING_ACTUALS = 'AWAITING HEARING ACTUALS',
```

Comment:
already done in https://github.com/hmcts/rpx-xui-webapp/pull/1773

Reply:
@paddybasi - this is now outdated as I merged the changes from base branch after you merged your changes (EUI-5092).